uwblueprint/jump-math
# add teacher table

## Notion ticket link

[Implement teacher users table](https://www.notion.so/uwblueprintexecs/Implement-Teacher-Users-table-eff0155d8e484d0dbcc5165a792482d0)



## Implementation description
* Followed general format of adminTable



## Steps to test
1. check that the data displayed is correct
2. check that searching/sorting works



## What should reviewers focus on?
* see comments


## Checklist
